stdlib/public/SDK/CoreFoundation/CoreFoundation.swift

```diff
@@ -0,0 +1,23 @@
+//===----------------------------------------------------------------------===//
```

Comment:
I would then prefer to see an empty CoreFoundation.swift and this stuff in a more descriptive place.

I really don't like how Foundation.swift became a dumping ground for whatever needed to be put there.